README.md

```diff
@@ -1 +1,61 @@
-# Oop-library
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# SChool Library
+> In this project i built a library database that will help the librarians at universities to record what books are in the library and who borrows them. 
+
+## Built With
+
+- RUBY
+- Linters
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+## To get a local copy up and running:
+
+1. Clone this repository or download the zip folder:
+
+**``https://github.com/School-library.git`**
+
+2. Navigate to the location of the folder in your machine:
+
+**``you@your-Pc-name:~$ cd <School-library>``**
+
+To get a local copy up and running follow these simple example steps above.
+
+
+### Prerequisites: 
+Any kind of browser. 
+
+### Setup:
+    1. Clone the repository (git clone https://School-library.git)
+    2. Then open the folder in your local environment where you cloned it.
+
+### Deployment
+
+[See project here]()
+
+## Authors
+
+👤 **Author**
+
+   - GitHub:[@markotuya](https://github.com/markotuya0)
+   - Twitter:[@markotuya](https://Twitter.com/mark_anthonny)
+   - LinkedIn:[@markotuya](https://www.linkedin.com/in/mark_anthonny)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/markotuya0/School-library/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+- Inspiration
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Your READEM looks professional 💯 But I noticed that The MIT link is broken, so I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯